src/core/Tooltip/style.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from "@emotion/css";
+import {
+  AppThemeOptions,
+  fontBodyXs,
+  getColors,
+  getShadows,
+  getSpacings,
+} from "../styles";
+
+export const tooltipCss = (theme: AppThemeOptions): string => {
+  const props = { theme };
+  const colors = getColors(props);
```

Comment:
since all the selectors expects a `props` interface, which is an object with key `theme`, we need to declare a `props` variable and pass it to all the selectors

Reply:
Oh interesting! If I also wanted the `Tooltip` component to take in some props that would affect styling, what would be the best way to go about doing that?

For context, Jenn had just reached out to me that they had updated some of the naming for the Tooltips which you can find [here](https://www.figma.com/file/EaRifXLFs54XTjO1Mlszkg/Science-Design-System-Reference?node-id=642%3A4116), but essentially `TooltipInfo` is now being consolidated into just `Tooltip` as its secondary style. `Tooltip`'s primary style will be inverted (black background, white text).

Reply:
Got it! What do you think of new prop `inverted` for that? 

From API's perspective, I feel "primary" and "secondary" are closely related to color. Like MUI's `color` prop typically accepts string `"primary"` and `"secondary"` to style the component with primary color or secondary color. So since what we're doing is not related to primary and secondary colors, I feel steering away from those terms would avoid confusion 😆  E.g., [Chip](https://material-ui.com/api/chip/#props)

As for accepting custom props, typically I use `interface ExtraProps { ... }` to capture them, and then update the component prop interface like so:

```ts
// src/core/ListItem/index.tsx

import { ListItemProps as RawListItemProps } from "@material-ui/core";
import { ExtraProps } from "./style";

export type ListItemProps = RawListItemProps & ExtraProps;

const ListItem = (props: ListItemProps): JSX.Element => {
  return <StyledListItem disableGutters {...(props as never)} />;
};
```

In our Tooltip case, it could be something like the following? :

```tsx
interface ExtraProps {
  inverted?: boolean;
}

type TooltipProps = ExtraProps & RawTooltipProps;

const TooltipInfo = (props: TooltipProps): JSX.Element => {
  const theme = useTheme();
  const tooltip = tooltipCss({ ...props, theme });
  const arrow = arrowCss({ ...props, theme });

  return <RawTooltip arrow classes={{ arrow, tooltip }} {...props} />;
};

TooltipInfo.defaultProps = {
  inverted: false,
};
```

Reply:
This looks great! Yeah, I'm also more accustomed to using `inverted` from semantic-ui as well. I'll ask Jenn what she thinks about changing it to inverted!